### PR TITLE
seslib: fix Ceph repos for ses5, ses6, ses7

### PR DESCRIPTION
**Pull request by smithfarm**
Fixes: https://github.com/SUSE/sesdev/issues/73
Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

File: seslib/__init__.py [hunk 1]
```diff
@@ -112,7 +112,8 @@ def init(cls, working_dir):
 VERSION_OS_REPO_MAPPING = {
     'ses5': {
         'sles-12-sp3': [
```
Comment (holgis) [inline seslib/__init__.py]: Is 'sles-12-sp3' still the right name?

File: seslib/__init__.py [hunk 2]
```diff
@@ -127,7 +128,8 @@ def init(cls, working_dir):
     },
     'ses6': {
         'sles-15-sp1': [
```
Comment (holgis) [inline seslib/__init__.py]: Is 'sles-15-sp1' still the right name?

File: seslib/__init__.py [hunk 3]
```diff
@@ -146,8 +148,8 @@ def init(cls, working_dir):
     },
     'ses7': {
         'sles-15-sp2': [
```
Comment (holgis) [inline seslib/__init__.py]: Is 'sles-15-sp2' still the right name?

File: seslib/__init__.py (hunk 1 above)
Comment (smithfarm) [reply seslib/__init__.py]: If the user does not specify the `--os` option, `sesdev create ses5` will use the os called "sles-12-sp3". This is not an official name -- rather, it's just a label that sesdev uses.

File: seslib/__init__.py (hunk 2 above)
Comment (smithfarm) [reply seslib/__init__.py]: If the user does not specify the `--os` option, `sesdev create ses6` will use the os called "sles-15-sp1". This is not an official name -- rather, it's just a label that sesdev uses.

File: seslib/__init__.py (hunk 3 above)
Comment (smithfarm) [reply seslib/__init__.py]: If the user does not specify the `--os` option, `sesdev create ses7` will use the os called "sles-15-sp2". This is not an official name -- rather, it's just a label that sesdev uses.

**Comment by smithfarm** [comment]
@holgis This PR is not changing the names of the dict keys. It is only changing the repos applied to the `ses[567]` OS targets.

**Pull request merged**